public/doc/swagger-2.yaml

```diff
@@ -4,18 +4,26 @@ info:
   version: 1.0.0
   title: Service Portal API
   contact:
-    email: you@your-company.com
+    email: support@redhat.com
   license:
     name: Apache 2.0
     url: 'http://www.apache.org/licenses/LICENSE-2.0.html'
 securityDefinitions:
-  UserSecurity:
-    type: basic
-  AdminSecurity:
+  BasicAuth:
```

Comment:
Is this a place holder for external access or is this a known requirement?

Reply:
@syncrou 
This is a known requirement the API can be accessed externally using BasicAuth by accessing access.redhat.com/r/h....
The other Auth Type is for internal access. The external access will return a 401